### PR TITLE
Updating to rxjs v6 in order to update to the latest angular cli

### PR DESCRIPTION
**Pull request by rachelrosengoogle**
These changes are needed in order to update conversationai-website to the newest versions of rxjs and angular-cli, which will hopefully fix a bug in conversationai-website.

### CODE REVIEW

**Comment by rachelrosengoogle** [comment]
Realized I never got this submitted. :)

File: src/polyfills.ts [hunk 1]
```diff
@@ -71,3 +71,7 @@ import 'hammerjs/hammer';
 // Needed on browsers with native `customElements`.
 // (E.g.: Chrome, Opera)
 import '@webcomponents/custom-elements/src/native-shim';
+
+// Add global to window, assigning the value of window itself.
+// See https://github.com/angular/angular-cli/issues/9827#issuecomment-386154063
+(window as any).global = window;
```
Comment (iislucas) [inline src/polyfills.ts]: Which library do we import that needs this? Add to comment?

Comment (rachelrosengoogle) [reply src/polyfills.ts]: Done

File: src/polyfills.ts [hunk 2]
```diff
@@ -71,3 +71,8 @@ import 'hammerjs/hammer';
 // Needed on browsers with native `customElements`.
 // (E.g.: Chrome, Opera)
 import '@webcomponents/custom-elements/src/native-shim';
+
+// Add global to window, assigning the value of window itself.
+// See https://github.com/angular/angular-cli/issues/9827#issuecomment-386154063
+// Needed for the twemoji library.
```
Comment (iislucas) [inline src/polyfills.ts]: Ah, cool, so this is important I think, and will probably have to be done by every project that includes this component. Want to add some info to the README.md on that? I think I found this and filed some issues related to it when I did the remake of the IO demo. Maybe for a followup PR

**Pull request merged**